avm/res/logic/workflow/main.bicep

```diff
@@ -62,7 +62,7 @@ param tags object?
 param triggersAccessControlConfiguration object?
 
 @description('Optional. The definitions for one or more actions to execute at workflow runtime.')
-param workflowActions object?
+param workflowActions object = {}
```

Comment:
Given this comment : https://github.com/Azure/bicep-registry-modules/issues/5124#issuecomment-2865694034

I'm not sure we want that to happen.

There is another alternative if both `null` and `{}` are not good ideas, and that is to implement it in a way where the property is not passed in at all if empty. This naturally has the caveat that you can not every remove all steps from the Logic App, as an empty object would just be ignored, but I'm not the expert here.

If you'd want to do that, you'd need to implement each property that should behave this way using the `shallowMerge` feature. 

For example
```bicep
workflowActions: ...(!empty(workflowActions) ? { actions: workflowActions } : {})
```

Reply:
@AlexanderSehr, I like the suggestion, however in testing it, if the property is not passed in, the logic app designer will not load. As such, the property must exist with an empty bracket {} if nothing is specified in the parameter. Isn't that expected behavior though? If I specify one of the properties as null, it should overwrite settings that may have been made in the portal? The deployment should reflect what is specified in the template.

Reply:
There are always two philosophies, but as you already validated it and one one works it makes the discussion easy 😄